docs/reference/multi-tenancy.md

```diff
@@ -0,0 +1,383 @@
+
+For example, cluster admins are able to apply namespaced and cluster-wide CRs. Team admins have the capability to deploy namespaced CRs within their designated namespaces. Meanwhile, team devs are restricted to view CRs within their allocated namespaces.
+
+### General Resources
+Users also need to restrict access to workload objects. For example, cluster admin should only allow access to pods, deployments, and services in a namespace from users assigned to the same namespace with correct permission.
+
+For example, a team admin could make a deployment to its namespace but fail deploying to a different namespace. And a team developer might only have access to view the deployment but not modify it.
+
+## Limitation
+This is the first version we introduced multi-tenancy support in Ratify. There are some limitations that users should be aware of:
+
+1. Ratify requires users to define cluster-wide Store CRs to mutate the image. We already created an [issue](https://github.com/open-policy-agent/gatekeeper/issues/3376) on Gatekeeper to support Ratify's user scenario. But the final plan would depend on GK's decision on this issue.
+2. Logs and metrics isolation is not fully supported yet. Users are responsible for assigning correct permissions to those who need accsss to logging and instrumenting platform. In an upcoming release, Ratify will introduce a namespace field for all logs and metrics, making filtering easier. We have issues tracking this 2 scenarios and will be addressed in next release. 
+https://github.com/deislabs/ratify/issues/1483
+https://github.com/deislabs/ratify/issues/1484
+
+## Setup
+To set up multi-tenancy in Ratify, users need to follow the steps below:
+
+### Step 1: Install Minikube
+In this tutorial, we take [Minikube](https://minikube.sigs.k8s.io/docs/) as an example, but feel free to choose your own Kubernetes Cluster.
+
+```bash
+curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+sudo install minikube-linux-amd64 /usr/local/bin/minikube && rm minikube-linux-amd64
+```
+
+### Step 2: Create a new team admin user besides the existing minikube cluster admin user.
+
+#### Step 2.1: Create a client certificate
+```bash
+mkdir cert && cd cert
+# Generate a key using OpenSSL
+openssl genrsa -out user1.key 2048
+# Generate a Client Sign Request
+openssl req -new -key user1.key -out user1.csr -subj "/CN=user1/O=group1"
+# Generate the certificate
+openssl x509 -req -in user1.csr -CA ~/.minikube/ca.crt -CAkey ~/.minikube/ca.key -CAcreateserial -out user1.crt -days 500
+```
+Now we have the `user1.key` and `user1.crt` to create a user.
+
+#### Step 2.2: Create a new user
+```bash
+# Set a user entry in kubeconfig
+kubectl config set-credentials user1 --client-certificate=user1.crt --client-key=user1.key
+# Set a context entry in kubeconfig
+kubectl config set-context user1-context --cluster=minikube --user=user1
```

Comment:
is user1 and teamAdmin? chould we name the user "teamadmin"?

Reply:
good call, will rename to team-admin